sql/util_time.sql

```diff
@@ -3,6 +3,12 @@
 -- LICENSE-APACHE for a copy of the license.
 
 -- This file contains utilities for time conversion.
+CREATE OR REPLACE FUNCTION _timescaledb_functions.get_min_for_type(REGTYPE) RETURNS BIGINT
```

Comment:
It is unclear what is being returned since BIGINT is a generic type. I suggest that the function name should be clear about what is being returned, which I guess is UNIX time?

Reply:
It is Unix time, yes. I added a comment about this, but since this is just returning limits for integer types (the Unix timestamp just makes sense for `timestamp` and `timestamptz`).